pkg/node/requester.go

```diff
@@ -238,6 +251,13 @@ func NewRequesterNode(
 		NodeDiscoverer:     nodeDiscoveryChain,
 	})
 
+	orchestrator_endpoint.NewEndpoint(orchestrator_endpoint.EndpointParams{
```

Comment:
the current pattern is these endpoints register themselves with the passed router and add their handlers. There is nothing needed from the returned instance. Maybe this can be improved